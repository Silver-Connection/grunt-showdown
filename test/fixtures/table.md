
|titre1          | titre2                | titre3       |
|----------------|-----------------------|--------------|
|valeur1         | valeur2               | valeur3      |

|漢字                             | かな        | フランス語で|
|--------------------------------|------------|------------|
|<font size="+3">起</font>       | おーきる・キ  | se lever    |
|<font size="+3">飲</font>       |　のーむ・イン   | boire      |
|<font size="+3">着</font>       | きーる・チャク  | mettre un vêtement |
|<font size="+3">遊</font>       |　あそーぶ・ユウ   | jouer             |
|<font size="+3">服</font>       | ふく         | vêtement             |
|<font size="+3">庭</font>       |　にわ・テイ   | jardin              |