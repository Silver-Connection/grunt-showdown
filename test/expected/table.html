<table>
<thead>
<tr>
<th id="titre1" style="text-align:left;">titre1          </th>
<th id="titre2" style="text-align:left;"> titre2                </th>
<th id="titre3" style="text-align:left;"> titre3       </th>
</tr>
</thead>

<tbody>
<tr>
<td style="text-align:left;"><p>valeur1         </p></td>
<td style="text-align:left;"><p>valeur2               </p></td>
<td style="text-align:left;"><p>valeur3      </p></td>
</tr>

</tbody>
</table>

<table>
<thead>
<tr>
<th id="titre1" style="text-align:left;">titre1          </th>
<th id="titre2" style="text-align:left;"> titre2                </th>
<th id="titre3" style="text-align:left;"> titre3       </th>
</tr>
</thead>

<tbody>
<tr>
<td style="text-align:left;"><p>valeur1         </p></td>
<td style="text-align:left;"><p>valeur2               </p></td>
<td style="text-align:left;"><p>valeur3      </p></td>
</tr>

</tbody>
</table>

<table>
<thead>
<tr>
<th id="漢字" style="text-align:left;">漢字                             </th>
<th id="かな" style="text-align:left;"> かな        </th>
<th id="フランス語で" style="text-align:left;"> フランス語で</th>
</tr>
</thead>

<tbody>
<tr>
<td style="text-align:left;"><p><font size="+3">起</font>       </p></td>
<td style="text-align:left;"><p>おーきる・キ  </p></td>
<td style="text-align:left;"><p>se lever    </p></td>
</tr>

<tr>
<td style="text-align:left;"><p><font size="+3">飲</font>       </p></td>
<td style="text-align:left;"><p>　のーむ・イン   </p></td>
<td style="text-align:left;"><p>boire      </p></td>
</tr>

<tr>
<td style="text-align:left;"><p><font size="+3">着</font>       </p></td>
<td style="text-align:left;"><p>きーる・チャク  </p></td>
<td style="text-align:left;"><p>mettre un vêtement </p></td>
</tr>

<tr>
<td style="text-align:left;"><p><font size="+3">遊</font>       </p></td>
<td style="text-align:left;"><p>　あそーぶ・ユウ   </p></td>
<td style="text-align:left;"><p>jouer             </p></td>
</tr>

<tr>
<td style="text-align:left;"><p><font size="+3">服</font>       </p></td>
<td style="text-align:left;"><p>ふく         </p></td>
<td style="text-align:left;"><p>vêtement             </p></td>
</tr>

<tr>
<td style="text-align:left;"><p><font size="+3">庭</font>       </p></td>
<td style="text-align:left;"><p>　にわ・テイ   </p></td>
<td style="text-align:left;"><p>jardin              </p></td>
</tr>

</tbody>
</table>